bitnami/postgresql-ha/Chart.yaml

```
@@ -27,4 +27,4 @@ name: postgresql-ha
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/postgresql
   - https://www.postgresql.org/
-version: 9.2.8
+version: 9.2.9
```

We are adding a new feature, so according to [semver](https://semver.org/), we should release the version `9.3.0`

```suggestion
version: 9.3.0
```